### PR TITLE
Applab: Header button styles

### PR DESCRIPTION
**Pull request by islemaster**
The "Start Over" and "Show Code" buttons get a design overhaul!

screenshot from 2015-06-17 10 38 16

In AppLab we've changed "Show Code" to "Show Text," since both the blocks and text are kinds of code.  They look more like buttons now and have associated icons.  Here's the mockup for comparison:

new_buttons_mockup

Clicking the "Show Text" button changes it to a "Show Blocks" button with its own icon.

screenshot from 2015-06-17 10 38 37

This button change is retroactive to our blockly tutorials as well; for those, we've left the "Show Code" text as-is.

screenshot from 2015-06-17 10 39 25

The buttons still have the rollover colors they had before this PR:

new_buttons_rollover_colors

We went back and forth about the </> icon for "Show Text" for quite a while, since we're not actually teaching any markup and it's a little bit of a misrepresentation (for the record, that is the fa-code icon for font-awesome).  We tried curly braces but the at a glance it just looked a bit like a circle:

show_code_test_01

Brendan suggested we use something like the "Show Blocks" icon but with twice as many lines - when we tried something similar it looked like the letter "E" at a glance, but we could give that another shot if people are really unhappy with </>

### CODE REVIEW

File: apps/src/StudioApp.js [hunk 1]
```diff
@@ -1530,9 +1530,28 @@ StudioApp.prototype.handleUsingBlockly_ = function (config) {
 StudioApp.prototype.updateHeadersAfterDropletToggle_ = function (usingBlocks) {
   // Update header titles:
   var showCodeHeader = document.getElementById('show-code-header');
-  var newButtonTitle = usingBlocks ? msg.showCodeHeader() :
-    msg.showBlocksHeader();
-  showCodeHeader.firstChild.textContent = newButtonTitle;
+  var contentSpan = showCodeHeader.firstChild;
+  var fontAwesomeGlyph = _.find(contentSpan.childNodes, function (node) {
+    return /\bfa\b/.test(node.className);
```
Comment (breville) [inline apps/src/StudioApp.js]: what does .test do?

**Comment by breville** [comment]
nice use of sass variables.  lgtm.

File: apps/src/StudioApp.js (hunk 1 above)
Comment (islemaster) [inline apps/src/StudioApp.js]: [RegExp.prototype.test()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test) checks whether its string argument matches the regular expression and returns `true` or `false`.  In this case, we're finding the first node with a className matching the regular expression `/\bfa\b/`, which is "fa" wrapped in word boundaries so we don't match something like "fall-colors".

**Comment by islemaster** [comment]
For the archives, here's a test of a four-line icon for "Show Text."

screenshot from 2015-06-17 11 33 49

I like this better than I expected, but it might be equally good as an icon for blocks mode as for text mode so we stuck with the </> glyph.

**Comment by islemaster** [comment]
Also for the archives, here's the RTL behavior:

new_buttons_rtl

**Pull request merged**